src/applications/mhv-secure-messaging/components/ComposeForm/RecipientsSelect.jsx

```diff
@@ -213,22 +217,41 @@ const RecipientsSelect = ({
 
   return (
     <>
-      <VaSelect
-        enable-analytics
-        id="recipient-dropdown"
-        label="To"
-        name="to"
-        value={defaultValue !== undefined ? defaultValue : ''}
-        onVaSelect={handleRecipientSelect}
-        class="composeSelect"
-        data-testid="compose-recipient-select"
-        error={error}
-        data-dd-privacy="mask"
-        data-dd-action-name="Compose Recipient Dropdown List"
-      >
-        <CantFindYourTeam />
-        {optionsValues}
-      </VaSelect>
+      {isComboBoxEnabled ? (
+        <VaComboBox
+          required
+          id="recipient-combobox"
+          label="Select a care team to send your message to"
```

Comment:
I see places in Figma where this and `To` label are both used. I am guessing this was confirmed with UCD?